### ISSUE TITLE
Avoid unnecessary force-unwrapping or nil-coalescing

### ISSUE THREAD

**Issue by blochberger** [corona-warn-app/cwa-app-ios#138]
There is a lot of unnecessary force-unwrapping or nil-coalescing in the code, e. g.:

https://github.com/corona-warn-app/cwa-app-ios/blob/e4a9d3263dc04b2b317a5ec92e40a941ca5985ac/src/xcode/ENA/ENA/Source/Developer%20Menu/DMSubmissionStateViewController.swift#L120

or

https://github.com/corona-warn-app/cwa-app-ios/blob/e4a9d3263dc04b2b317a5ec92e40a941ca5985ac/src/xcode/ENA/ENA/Source/Workers/Hasher.swift#L24

The conversion from `String` to UTF-8 encoded `Data` cannot fail. A non-optional conversion can be done:

```swift
let string: String = "foobar"
let data: Data = Data(string.utf8)
```

**Comment by tkowark** [comment]
Thanks for the enhancement proposal, @blochberger !

As the team is working full steam towards the planned release, it would help us a lot, if you could give us some more information about the potential benefits of your proposal. Ideally, you can follow (and in the future use) the [respective template](https://github.com/corona-warn-app/cwa-app-ios/issues/new?labels=enhancement&template=03_enhancement.md).

The background simply is that we have to pick wisely, which improvements to implement now, and which ones to tackle in a future release. To better assess that, just a rough outline of the expected benefits (readability, error-proneness, performance, etc.) would really be of huge assistance!

**Comment by ChristianKienle** [comment]
will be fixed by #402. Thank you very much for pointing out those unnecessary forced unwraps / default values.
